Lib/test/test_importlib/test_metadata_api.py

```diff
@@ -172,6 +172,10 @@ def test_entry_points_groups_get(self):
             entry_points().get('entries', 'default') == entry_points()['entries']
             entry_points().get('missing', ()) == ()
 
+    def test_entry_points_allows_no_attributes(self):
+        with self.assertRaises(AttributeError):
+            entry_points()['entries'][0].foo = 4
```

Comment:
This test uses a deprecated interface on SelectableGroups. I'm surprised the tests passed with a deprecation.

Reply:
Interestingly, the test doesn't fail either prior to adding the fix.

Reply:
Fixed the test in 16d66e1de5.